packages/vite/src/node/plugins/index.ts

```diff
@@ -1,7 +1,7 @@
 import { ResolvedConfig } from '../config'
 import { Plugin } from '../plugin'
 import aliasPlugin from '@rollup/plugin-alias'
-import jsonPlugin from '@rollup/plugin-json'
```

Comment:
Should remove `@rollup/plugin-json` from deps

Reply:
https://github.com/vitejs/vite/blob/91dbb017091c175a54bcd1c93a69f8458d1bde8d/packages/vite/rollup.config.js#L8
Is also using it so i didn't remove it,
should I change it to use customized json plugin too?

Reply:
Ah ok, let's keep it as-is then.